### PR TITLE
Add data science minor

### PR DESCRIPTION
**Pull request by elizabeth-tang**
**Add data science minor to minors.** 

This is a new minor offered starting Spring 2023: https://stat.cornell.edu/academics/undergraduate/data-science-minor/minor-requirements

### Test Plan 

- Test if ENGRD 2700 or ECE 3100 fulfill core statistics requirement.
- Test if the two subrequirements in "Categories" requirement work as expected - 4 total courses in 3 categories.

### Notes 

- There is no existing way to check against classes taken for a student's major.
   - Applies to computing requirement. If intro computing is required for major, then they cannot use intro computing for the req. Currently addressed as a checkerWarning.
   - Applies to Categories requirement. Courses cannot also be used in major core. Currently included in just the description as checkerWarning produces user-inconveniencing behavior.
- The minor is technically offered under the CIS school.
- No set information available on advisors yet. I tentatively included the undergraduate director for data science as the advisor.

### CODE REVIEW

**Comment by noschiff** [comment]
make sure to run `npm run req-gen` and `npm test -- -u` after you make any changes to the requirements file :)
once you get all the CI/CD checks passing, we can review! 🎉

**Comment by noschiff** [comment]
> There is no existing way to check against classes taken for a student's major.

@zachary0kent what's our plan to address this? I remember @rohanmaheshwari430 brought something similar up?

**Comment by zachary0kent** [comment]
> > There is no existing way to check against classes taken for a student's major.
> 
> @zachary0kent what's our plan to address this? I remember @rohanmaheshwari430 brought something similar up?

What Rohan had discussed is different, a "4 courses comprising at least 18 credits" requirement

**Review by zachary0kent — COMMENTED** [review]
Looks good! Is there a reason why some requirements allow double counting?

File: src/data/index.ts [hunk 1]
```diff
@@ -415,6 +416,12 @@ const json: RequirementsJson = {
       requirements: csMinorRequirements,
       advisors: csMinorAdvisors,
     },
+    DS: {
```
Comment (noschiff) [inline src/data/index.ts:419]: let's merge master into this branch and add the abbreviation

**Pull request closed without merging**